src/js/demo/components/Configuration.jsx

```diff
@@ -30,6 +30,11 @@ export default class Configuration extends Component {
         }
     }
 
+    shouldComponentUpdate(nextProps, nextState) {
+        return this.state.showingConfig !== nextState.showingConfig ||
+            JSON.stringify(this.props.options) !== JSON.stringify(nextProps.options);
```

Comment:
I’d have to dig through the code a bit, but is there any chance the ordering of properties could change?

Reply:
It does change all the time while the user is changing configuration, but that's when the component should indeed update. Looks like the order doesn't change in other situations, and this solutions seem to be OK.

But I'd like to try something else instead - the references also don't change and there are no mutations in props, so it seems that `PureComponent` should also do the same here.